### DISCUSSION TITLE
Is it possible to center the summary only?

### DISCUSSION THREAD

**Discussion by randomcodepanda** [dunst-project/dunst#1354] [Help]
I'm just checking if it's possible to do this, as i would like to center only the summary to kind of look like a title.

There is an option "alignment" but that centers all the text.

**Comment by bynect** [comment]
Maybe you could try with the pango format? It should support some html tags. See `format`

**Reply by randomcodepanda** [reply]
I've tried with < center > tags and also with style="text-align:center" and it doesn't seem to work, i don't know if there are any other ways.

What i'm thinking now is that if i use a fixed width then when sending the summary i could fill each side of the text with whitespace so it's centered, although it's a somewhat hacky solution, ideally you would set the width when sending the notification but i don't think that's possible.
